activerecord/lib/active_record/model_schema.rb

```diff
@@ -289,6 +289,7 @@ def ignored_columns
       # accessors defined, and won't be referenced in SQL queries.
       def ignored_columns=(columns)
         @ignored_columns = columns.map(&:to_s)
+        reload_schema_from_cache if schema_loaded?
```

Comment:
Can you move this line above the other one? It's conventional for setters to return the value that was just set.

You can also ✂️ the `if schema_loaded?` check; it's not a particularly expensive method.

This matches how `reload_schema_from_cache` is used in `locking_column=`:

https://github.com/rails/rails/blob/d5a3b2e427046b48aefe6f634b1c6ceb33e5c1bc/activerecord/lib/active_record/locking/optimistic.rb#L135-L138

Reply:
Nice! Thanks a lot for the feedback with clear instructions and a reference to another example from the codebase 🤗that helps us a lot with getting started. I'll submit the changes asap.